cirq/__init__.py

```diff
@@ -47,6 +47,7 @@
     ion,
     neutral_atoms,
     google,
+    iqm,
```

Comment:
We may want to consider putting this in contrib instead of a top-level package.

@Strilanc Any thoughts?

Reply:
I agree.

Reply:
contrib has the danger that it hides the importance of direct real quant hardware access. this code, as well as cirq.aqt, for example, will offer the possibility to run circuits on different hardware platforms.

Reply:
contrib has the danger that it hides the importance of direct real quant hardware access. this code, as well as cirq.aqt, for example, will offer the possibility to run circuits on different hardware platforms.